src/components/habitEditor/habitEditorComponents/HabitEditorScore.js

```diff
@@ -0,0 +1,100 @@
+import { useState } from "react";
+import styled from "styled-components";
+import ScoreLineChart from "./charts/ScoreLineChart";
+
+const HabitEditorScore = ({ habitCheckList, getPrevDate }) => {
+	const [graphState, setGraphState] = useState("day");
+	const handleGraphChange = (e) => {
+		setGraphState(e.target.value);
+	};
+	let graphInterval = 1;
+	switch (graphState) {
+		case "day":
```

Comment:
I think this can be handled cleanly creation a mapping object e.g. ` {day:1, week:7 etc}`